src/harness/harnessLanguageService.ts

```diff
@@ -333,6 +333,9 @@ module Harness.LanguageService {
         getDefinitionAtPosition(fileName: string, position: number): ts.DefinitionInfo[] {
             return unwrapJSONCallResult(this.shim.getDefinitionAtPosition(fileName, position));
         }
+        getTypeDefinitionAtPosition(fileName: string, position: number): ts.DefinitionInfo[]{

```

Comment:
That makes sense.  We shoudl drive this through on the Roslyn side as well though.  Definitely open up a bug on them to support this.  That way we can get a command in for VB/C#/TS for this scenario, and the experience can be consistent between all three languages.